.travis.yml

```diff
@@ -29,6 +34,24 @@ script:
 after_success:
     - coveralls
 
+before_deploy:
+    - sudo add-apt-repository -y ppa:spotify-jyrki/dh-virtualenv
+    - sudo apt-get update
+    - sudo apt-get install -y devscripts python-virtualenv git equivs dh-virtualenv
+    - make build-package
+
+deploy:
+    -   provider: script
+        script: scripts/travis_deploy_docker.sh
+        on:
+            branch: ng
+            python: 3.4
```

Comment:
Docker image and DEB package are independent from python version on travis. Python version to use there should be specified either in Dockerfile or `debian/rules`. 

This condition is there only for one reason: to not deploy anything multiple times (in many builds on travis).